import-automation/progress-dashboard-rest/test/utils.py

```diff
@@ -0,0 +1,112 @@
+import os
```

Comment:
Do we need to update test/requirements.txt?

Reply:
For which modules?

Reply:
Added google-auth just in case